resources/public/api/conf/2.0/application.yaml

```diff
@@ -1388,21 +1374,6 @@ components:
         message:
           type: string
           description: The description of the error.
-    Conflict:
-      title: Conflict
-      description: LRN has previously been used and cannot be reused.
-      required:
-        - code
-        - message
-      type: object
-      properties:
-        code:
-          enum:
-            - CONFLICT
-          type: string
-        message:
-          type: string
-          description: The description of the error.

```

Comment:
Did we miss this for 11 October comms and content?

Reply:
No, this bit is just the LRN reuse stuff so it's more supplementary to it - I mentioned that the 409 is no longer sent on a duplicate LRN, but an IE056 is. I just forgot to change it here.

Reply:
OK.